integration-tests/tests/test_utils.rs

```diff
@@ -382,7 +384,8 @@ where
         .host(&target)
         .target(&target)
         .opt_level(1)
-        .flag("-std=c++14");
+        .flag("-std=c++14") // For clang
```

Comment:
Should this be `flag_if_supported`? It seems surprising this works! Obviously, it does though...

Reply:
Raied https://github.com/google/autocxx/issues/824